### ISSUE TITLE
Model not converging due to high loss from QUA

### ISSUE THREAD

**Issue by pranamyapatil** [dqqcasia/mosst#4]
### Before asking:   
1. search the issues.   
2. search the docs.    

#### What have you tried?
I tried to train model for supporting adaptive wait k. When i was training the model I am getting very high loss from length penalty ie:- qua component and the loss doesn't converge. Any idea of what can be the issue. Also while calculating final loss why are we multiplying qua loss with ntokens/nsamples.

#### What's your environment?

 - fairseq Version (e.g., 1.0 or master):
 - PyTorch Version (e.g., 1.0)
 - OS (e.g., Linux):
 - How you installed fairseq (`pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:
